frontend/src/components/Projects.js

```diff
@@ -1,62 +1,67 @@
-        <div className="inner">
-            <img className="card-img-top rounded" src={image} alt=""/>
-        </div> 
-        <div className="card-body">
-          <div className="row">
-            <h6 className="card-title text-primary fw-bold">{title}</h6>
-            <p className="card-text text-secondary fw-bold">[{date}]</p>
+    <Link style={{textDecoration: 'none'}} to={`${slug}`}>
+      <div className="cursor-pointer">
+        <div className="card border-0">
+          <div className="inner">
+              <img className="card-img-top rounded" src={imgUrl} alt=""/>
+          </div> 
+          <div>
+            <div className="row">
+              <h6 className="card-title text-primary fw-bold mt-3">{title}</h6>
+              <div className="card-text text-muted text-secondary fst-italic">
+                <small class="text-muted">
+                  {date}
+                </small>
+              </div>
+            </div>
           </div>
         </div>
       </div>
-    </div>
+    </Link>
   )
 }
 
-// const SeeMore = ({link}) => {
-//   return (
-//     <div className="text-end fst-italic text-primary mb-5">
-//       <a href={link}>See more</a>
-//     </div>
-//   )
-// }
-
 const Projects = () => {
-  return (
-    <div className="">
-      {/* PROJECT */}
-      <div class="container">
-        <SectionHeader text="Projects" />
-        <div className="row">
-          <div className="col-sm">
-            <Card image={ dummy } title='Landscape Research Project' date='Present' />
-          </div>
-          <div className="col-sm">
-            <Card image={ dummy } title='STEM Club Website' date='Past' />
-          </div>
-          <div className="col-sm">
-            <Card image={ dummy } title='STEM Club Website' date='Past' />
-          </div>
-        </div>
+  const [projects, setProjects] = useState([])
+  useEffect(() => {
+    const fetchProjects = async () => {
+      const res = await axios.get(`${BACKEND_URL}/api/projects?populate=*`)
+      console.log(`${BACKEND_URL}/api/projects?populate=*`);
+      console.log(res.data.data);
+      setProjects(res.data.data)
+    }
 
-        <div className="row">
-          <div className="col-sm">
-            <Card image={ dummy } title='SQL Workshop Series' date='Upcoming' />
-          </div>
-          <div className="col-sm">
-            <Card image={ dummy } title='STEMazine' date='Past'/>
-          </div>
-          <div className="col-sm">
-            <Card image={ dummy } title='STEM Club Website' date='Past' />
+    fetchProjects()
+  }, [])
+
+  return (
+    <div className="container pt-4 col-9">
+      <SectionHeader text="Projects" />
+      <div className="row my-5">
+        {projects
+        .map(({attributes}) => 
+          <div class="col-md-4 mb-5">
+            <Card 
+              image={ attributes.featureImage } 
+              title={ attributes.name } 
+              date={ moment().startOf('day').fromNow() } 
```

Comment:
Is this a hack? Looks like we are measuring distance from the start of the date?

Reply:
I just updated the code for this part, please review